packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/home/index.js

```diff
@@ -2,21 +2,25 @@ import angular from 'angular';
 import '@uirouter/angularjs';
 import 'oclazyload';
 
+import '@ovh-ux/manager-core';
+
 const moduleName = 'ovhManagerPciAIDashboardHomeLazyLoading';
 
-angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
-  /* @ngInject */ ($stateProvider) => {
-    $stateProvider.state('pci.projects.project.ai-dashboard.home.**', {
-      url: '/home',
-      lazyLoad: ($transition$) => {
-        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+angular
+  .module(moduleName, ['ui.router', 'ovhManagerCore', 'oc.lazyLoad'])
```

Comment:
**question:** What is the need for including `ovhManagerCore` in this module instead of `home.module.js`?

Reply:
We actually don't need it, removed here: https://github.com/ovh/manager/pull/8172/commits/ede67bf06474e79c5ebd56df9418fdc3266b343d